pom_confluent.xml

```diff
@@ -44,8 +44,8 @@
 
     <!-- Set our Language Level to Java 8 -->
     <properties>
-        <maven.compiler.source>1.8</maven.compiler.source>
-        <maven.compiler.target>1.8</maven.compiler.target>
+        <maven.compiler.source>17</maven.compiler.source>
+        <maven.compiler.target>17</maven.compiler.target>
```

Comment:
@acristu this is because of some issue on your local trying to build, right? Couldn't see the change in Snowflake's repository.

Reply:
yeah, this is introduced by me